LegendaryExplorer/LegendaryExplorer/Tools/PackageEditor/Experiments/PackageEditorExperimentsO.cs

```diff
@@ -433,5 +435,38 @@ public static void DumpTOC()
             var toc = new TOCBinFile(inputFile);
             toc.DumpTOCToTxtFile(outputFile);
         }
+
+        public static void AddPresetGroup(string preset, PackageEditorWindow pew)
```

Comment:
This should check for pew.SelectedItem != null and pew.SelectedItem.Entry != null (the root object will have a null entry).